docs/red_core_data_statement.rst

```diff
@@ -0,0 +1,76 @@
+.. Red Core Data Statement
+
+=====================
+Red and End User Data
+=====================
+
+Notes for everyone
+******************
+
+What data Red collects
+----------------------
+
+Red, and the cogs included with it collect some amount of data
+about users for it's normal operations. 
+
+In particular, the bot will keep track of a short history of usernames/nicknames,
+which actions refer to your discord account (such as creating a playlist)
+as well as the content of specific messages used directly as commands for the bot
+(such as reports sent to servers).
+
+By default, Red will not collect any more data than it needs, and will not use it
+for anything other than the functionality of the bot that required it.
+
+3rd party extensions may store additional things beyond this. You can use the command
+``[p]mydata 3rdparty`` to view statements about this made by the authors of 
+the specific 3rd party extensions an instance of Red has installed.
+
+How can I delete data Red has about me?
+---------------------------------------
+
+The command ``[p]mydata forgetme`` provides a way for users to remove
+large portions of their own data from the bot. This command will not
+remove operational data, such as a record that your
+discord account was the target of a moderation action.
+
+3rd party extensions to Red are able to delete data when this command
+is used as well, but this is something each extension must implement.
+
+Additional Notes for Bot Owners and Hosts
+*****************************************
+
+How to comply with a request from Discord Trust & Safety
+--------------------------------------------------------
+
+There are a handful of these available to bot owners in the command group
+``[p]mydata ownermanagement``
+
+The most pertinent one if asked to delete data by a member of Trust & Safety
+is
+
+``[p]mydata ownermanagement processdiscordrequest``
+
+which will more completely remove a user.
```

Comment:
Sounds wrong, not sure what to change it to. 

which will completely purge a user may be one.